src/main/java/wanted/preonboarding/backend/dto/request/ItemSaveRequest.java

```diff
@@ -0,0 +1,11 @@
+package wanted.preonboarding.backend.dto.request;
+
+import lombok.Getter;
+
+@Getter
+public class ItemSaveRequest {
```

Comment:
Record를 사용하면 객체의 불변성을 조금 더 확실하게 보장할 수 있을 것 같아요!
```suggestion
public record ItemSaveRequest(String name, int price, int stock) {}

Reply:
훨씬 간결하고 확실하네요. 적극 활용해보겠습니다.